templates/plans/index.hamlet

```diff
@@ -0,0 +1,13 @@
+$forall plan <- plans
+  <div .plan .col-md-4>
+    <div .panel.panel-default>
+      <div .panel-heading>
+        <h3 .panel-title>#{show plan}
+      <div .panel-body>
+        ^{planFeatures plan}
+
+<h2>Why Pay?
+
+<p>Carnival will never contain ads
+<p>Carnival will never track your behavior
```

Comment:
This is sort of a lofty claim. Does this prevent us from using Intercom or Mixpanel? We might want to be more specific about what we won't do here.